src/pages/SignUp.js

```diff
@@ -89,19 +282,53 @@ const FormStyled = styled.form`
 `
 
 const InputStyled = styled.input`
-  height: 40px;
+  height: 36px;
+  width: 98%;
   border: 0;
   border-radius: 2px;
   padding: 4px;
   font-family: sans-serif;
   font-size: 14px;
   font-weight: 100;
-  box-shadow: 0 1px 0 0 rgba(0, 0, 0, 0.2);
-  color: #414756;
-  background: #aed6dc;
-  border: none;
-  border-bottom: 1px solid #414756;
-  margin-bottom: 28px;
+  margin: 12px auto 12px 4px;
+  box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.1);
+  background-color: #d8f7fc;
+  color: #353b40;
+`
+const SelectStyled = styled.select`
+  height: 36px;
+  width: 98%;
+  border: 0;
+  border-radius: 2px;
+  padding: 4px;
+  font-family: sans-serif;
+  font-size: 14px;
+  font-weight: 100;
+  margin: 12px auto 12px 4px;
+  box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.1);
+  background-color: #d8f7fc;
+  color: #353b40;
+`
+const LongInputStyled = styled.textarea`
```

Comment:
ich hab das über eine Klasse in GlobalStyles gelöst und dann nur die Elemente behalten, die sich unterscheiden...